manageiq-providers-vmware.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "fog-vcloud-director",    "~> 0.3.0"
   spec.add_dependency "ffi-vix_disk_lib",       "~>1.1"
-  spec.add_dependency "rbvmomi",                "~>3.0"
+  spec.add_dependency "rbvmomi2",               "~>3.0"
   spec.add_dependency "vmware_web_service",     "~>2.1.0"
```

Comment:
`rbvmomi` was removed from `vmware_web_service` in v2.0.0 (https://github.com/ManageIQ/vmware_web_service/commit/e558246699ae2c2b3f8a6196bed91e03cfa9870c) so no need to do anything else here